RedditSharp/Reddit.cs

```diff
@@ -110,12 +121,17 @@ public Reddit(DefaultWebAgent.RateLimitMode limitMode, bool useSsl = true)
         /// <param name="username"></param>
         /// <param name="password"></param>
         /// <param name="useSsl"></param>
+        [Obsolete]
```

Comment:
oops - I meant to suggest that as a separate commit.

Deprecated - obsolete.  Pretty much the same thing.

from the MSDN docs 

> Marking an element as obsolete informs users that the element will be removed in future versions of the product.

https://msdn.microsoft.com/en-us/library/system.obsoleteattribute(v=vs.110).aspx

either way it should be 

```csharp
[Obsolete("OAuth is recommended.", false)] 
```